src/analysis/utils/disk-utils.ts

```diff
@@ -46,7 +46,7 @@ export const defaultDataprocMasterDiskSize = 150;
 export const defaultDataprocWorkerDiskSize = 150;
 // Since Leonardo started supporting persistent disks (PDs) for GCE VMs, boot disk size for a GCE VM
 // with a PD has been non-user-customizable. Terra UI uses the value below for cost estimate calculations only.
-export const defaultGceBootDiskSize = 120;
```

Comment:
The cached GCE image is 120Gb already, so maybe we need to split between the case when we use cached terra-docker images, vs non-cached custom images

Reply:
Could we have a call on this? I might not understand the situation or the best action to take from here.

Reply:
We can parking lot :) But If you test on your BEE, I suspect that the boot disk size of the VM lunched with a cached terra-docker image (e.g. the default GATK one) will remain at 120Gb, but I could be wrong

Reply:
That sounds like a win then! Maybe no action needed in that case.

Reply:
We might need to make a change? If the custom image are using a 250Gb disk, but the cached images are using a 120GB disk, then the cost display might need to be a bit different

Reply:
I see, they're encoded in the `gce_image.wf.json` and `create_dataproc_image.sh` files, and the Jenkinsfile. I can't claim to understand everything happening here but that makes sense! I'll update when my test is done